cmd/minishift/cmd/start_preflight.go

```diff
@@ -37,23 +36,25 @@ import (
 	"github.com/minishift/minishift/pkg/util/os/atexit"
 	"github.com/spf13/viper"
 
-	cmdUtils "github.com/minishift/minishift/cmd/minishift/cmd/util"
+	cmdUtil "github.com/minishift/minishift/cmd/minishift/cmd/util"
```

Comment:
This was inconsistently named with how it was imported everywhere else in the project and so I just fixed it here.

Reply:
OK